app/lib/actions/katello/repository/upload_files.rb

```diff
@@ -5,6 +5,8 @@ module Actions
   module Katello
     module Repository
       class UploadFiles < Actions::EntryAction
+        middleware.use Actions::Middleware::KeepCurrentUser
```

Comment:
this isn't needed anymore after https://github.com/theforeman/foreman-tasks/commit/cf5c3a8b9fbe5f0b853341531d023e066af88c4f

The middleware gets added to all actions.

Reply:
Thanks for info, will remove. Patch was tested on some older katello version where this was proven as necessary.